src/test/crimson/seastore/test_object_data_handler.cc

```diff
@@ -480,6 +540,142 @@ TEST_P(object_data_handler_test_t, multiple_split) {
   });
 }
 
+struct object_data_handler_rbm_overwrite_test_t :
+  object_data_handler_test_t {
+  
+  object_data_handler_rbm_overwrite_test_t() : gen(rd()) {}
+
+  void set_overwrite() {
+    crimson::common::local_conf().set_val("seastore_data_delta_based_overwrite", "131072").get();
+  }
+
+  laddr_t get_random_laddr(size_t block_size, laddr_t limit) {
+    return block_size *
+      std::uniform_int_distribution<>(0, (limit / block_size) - 1)(gen);
+  }
+
+  TransactionRef create_transaction() {
+    return create_mutate_transaction();
+  }
+  std::random_device rd;
+  std::mt19937 gen;
+};
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, multi_write)
+{
+  run_async([this] {
+    set_overwrite();
+    test_multi_write();
+  });
+}
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, write_hole)
+{
+  run_async([this] {
+    set_overwrite();
+    test_write_hole();
+  });
+}
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, overwrite_single)
+{
+  run_async([this] {
+    set_overwrite();
+    test_overwrite_single();
+  });
+}
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, overwrite_double)
+{
+  run_async([this] {
+    set_overwrite();
+    test_overwrite_double();
+  });
+}
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, overwrite_partial)
+{
+  run_async([this] {
+    set_overwrite();
+    test_overwrite_partial();
+  });
+}
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, unaligned_write)
+{
+  run_async([this] {
+    set_overwrite();
+    test_unaligned_write();
+  });
+}
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, unaligned_overwrite)
+{
+  run_async([this] {
+    set_overwrite();
+    test_unaligned_overwrite();
+  });
+}
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, truncate)
+{
+  run_async([this] {
+    set_overwrite();
+    test_truncate();
+  });
+}
+
+TEST_P(object_data_handler_rbm_overwrite_test_t, no_split) {
+  run_async([this] {
+    set_overwrite();
+    test_no_split();
+    auto pins = get_mappings(0, 8<<10);
+    EXPECT_EQ(pins.size(), 1);
+    read(0, 8<<10);
+  });
+}
```

Comment:
See the previous comment, since the tests (and the below) are not rbm specific, can `object_data_handler_test_t` be reused instead of another `object_data_handler_rbm_overwrite_test_t` class ?

Also, the related test case names can be adjusted, for example:
* `no_split` rename to `no_remap` and `no_overwrite`, with and without `set_overwrite_limit()`;
* `split_left` rename to `remap_left` and `overwrite_write`, with and without  `set_overwrite_limit()`;
* ...

Reply:
Also rename `set_overwrite()` as `set_overwrite_threshold()` to be more explicit.